### ISSUE TITLE
Aidbox references on extensions when using FHIR endpoint

### ISSUE THREAD

**Issue by yosbeleg89** [Aidbox/Issues#329]
We created an extension like:

```
id: Practitioner.signature
type: { id: Signature, resourceType: Entity }
resourceType: Attribute
module: my-module
resource: { id: Practitioner, resourceType: Entity }
path: [signature]
description: Practitioner signature
extensionUrl: http://...
```

But when using the fhir endpoint we are receiving references in Aidbox format:
```
GET /fhir/Practitioner/pract1

...
resourceType: Practitioner
extension:
- url: http://...
  valueSignature:
    who: {id: role1, resourceType: PractitionerRole}
...
```

**Comment by niquola** [comment]
Do you mean ```type: { id: Signature, resourceType: Reference <---```

**Comment by niquola** [comment]
Show me the whole extension definition

**Comment by niquola** [comment]
Fixed in 0.5.0-SNAPSHOT; will be available in 10 min after build
